package.json

```diff
@@ -16,8 +16,8 @@
     "name": "Kristian Dupont",
     "url": "http://www.kristiandupont.com"
   },
-  "main": "build/engine.js",
-  "types": "build/engine.d.ts",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
```

Comment:
Changed "main" and "types" here to export the `Rule` and the `Config` types from the package root. The `processDatabase` function is exported as before.